src/legacy/core_plugins/vega/index.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { resolve } from 'path';
+import { Legacy } from 'kibana';
+
+const vegaPluginInitializer = (kibana: any) =>
+  new kibana.Plugin({
+    id: 'vega',
+    require: ['kibana', 'elasticsearch', 'visualizations', 'data'],
```

Comment:
Eventually Interpreter will live in `data` plugin, but currently it is in its own `interpreter` legacy plugin. So, you need to include `interpreter` here. Also, probably leave `data` here, too, for future-proofing.

Reply:
done